docs/content/doc/installation/with-docker.en-us.md

```diff
@@ -209,6 +217,8 @@ services:
     volumes:
 -      - ./gitea:/data
 +      - gitea:/data
```

Comment:
Why did you change this?

Reply:
It’s not changed. The docs contain a code block for diff type and so GitHub renders the diff. If you “view file” rendered you’ll see the docs are recommending a change for users to make to their compose file.